packages/vxrn/src/create.ts

```diff
@@ -529,12 +529,18 @@ export const create = async (optionsIn: VXRNConfig) => {
   // TODO move these to router.get():
   app.use(
     defineEventHandler(async ({ node: { req } }) => {
-      if (!req.headers['user-agent']?.match(/Expo|React/)) {
```

Comment:
btw wont this break now for RN vanilla? we may need both or something.

Reply:
no, because bare RN at beginning makes request to:
```
 → /status
 → /index.bundle
```
so this change won't bare